tests/unit/ResolutionDates.spec.ts

```diff
@@ -92,7 +101,7 @@ describe('Resolution Dates component - edit mode', () => {
   })
 
   it('displays previous dates', async () => {
-    const wrapper = wrapperFactory({ previousDates: arrayOfDates })
+    const wrapper = wrapperFactory({ previousDates: previousDates })
```

Comment:
Can be abbreviated to `{ previousDates }` if you like.

Reply:
oo nice trick. Will do!

Reply:
Updated in next commit 👍